yarn-project/circuits.js/src/cbind/circuits.gen.ts

```diff
@@ -33,7 +33,7 @@ import {
   PublicKernelInputs,
   CircuitError,
   isCircuitError,
-  ProverBasePtr,
+  RawPointerProverBase,
```

Comment:
This name was the one automatically generated in the code refactor, seemed good enough